lalrpop/src/build/action.rs

```diff
@@ -92,7 +92,10 @@ fn emit_user_action_code<W: Write>(grammar: &r::Grammar,
                                                   .iter()
                                                   .cloned()
                                                   .map(|t| grammar.types.spanned_type(t)))
-                                         .map(|(p, t)| format!("(_, {}, _): {}", p, t))
+                                         .map(|(p, t)| format!("(_, {}{}, _): {}",
```

Comment:
Perhaps `((m, p), t)` would be a bit clearer?

Reply:
I'm considering making a struct that contains the 'Pattern', with the name and whether it's mutable. I'd like some feedback on whether that would be useful or just confuse things more.